### DISCUSSION TITLE
OPT-200-Organization-Company wrong position in XML structure

### DISCUSSION THREAD

**Discussion by user6330** [OP-TED/eForms-SDK#426] [Q&A] [answered]
Hello,

Looking at the notice-editor demo, when I compile this
![image](https://user-images.githubusercontent.com/127392893/233946306-4d58d286-a1b6-42ba-a938-7241d37369c7.png)

It returns in the Physical Model a.k.a XML structure as
![image](https://user-images.githubusercontent.com/127392893/233946752-aa0a37f5-478a-4cb6-bba7-ea5870927fb6.png)

But looking at the fields.json, the xpathAbsolute is
**/*/ext:UBLExtensions/ext:UBLExtension/ext:ExtensionContent/efext:EformsExtension/efac:Organizations/efac:Organization/efac:Company/cac:PartyIdentification/cbc:ID**

so it must be under _ext:UBLExtensions/ext:UBLExtension/_.. and not under root.

I've tried some custom implementation, and I arrived to this structure
![image](https://user-images.githubusercontent.com/127392893/233947494-b1858349-62c8-41e4-b893-bceac651bdcb.png)

But when I try the asserts condition to validate, I found out that is missing
**efac:Organizations**, parent of **efac:Organization**.

Is It possible that the problem it's in the missing "nodeId" in hierarchy of the field "" inside the notice-subtype .json (in my case 1.json, but I think all the other notice-subtype)?

{
    "id" : "GR-Organisations-Section",
    "contentType" : "group",
    "displayType" : "SECTION",
    "description" : "Organisations",
    "_label" : "group|name|GR-Organisations-Section",
    "content" : [ {
      "id" : "GR-Organisations-Subsection",
      "contentType" : "group",
      "displayType" : "SECTION",
      "description" : "Organisations",
      "_label" : "group|name|GR-Organisations-Subsection",
      "content" : [ {
        "id" : "GR-Organisations",
        "contentType" : "group",
        "nodeId" : "ND-Organization",
        "displayType" : "GROUP",
        "description" : "A Party involved in the Competition from either side (Buyer or Tenderer) with a Legal Entity (Company) and possibly additional Contact Points (touchpoints)",
        "_label" : "group|name|ND-Organization",
        "_idScheme" : "ORG",
        "_schemeName" : "organization",
        "_identifierFieldId" : "OPT-200-Organization-Company",
        "_captionFieldId" : "BT-500-Organization-Company",
        "_repeatable" : true,
        "content" : [ {
          "id" : "OPT-200-Organization-Company",
          "contentType" : "field",
          "displayType" : "TEXTBOX",
          "description" : "Organisation Technical Identifier",
          "_label" : "field|name|OPT-200-Organization-Company",
          "_idScheme" : "ORG",
          "_schemeName" : "organization",
          "hidden" : true
        }
.............

![image](https://user-images.githubusercontent.com/127392893/233949362-b059c770-2a4d-4a56-8951-99dbdb786057.png)

**Comment by elerik** [comment, marked as answer]
Hi 
well check xmlStructure in field.json. This might to help you and eventualy you will get correct xml nodes. 
Anyway 1.json or X.json I avoid them for this. What i understand its more like visual map for us how to provide fields to our users in logic way and how to visualize requested data (for that they are pretty good). There can be differences (now or in future) for final xml (i think thay call it physical model)

**Comment by AlexAS81** [comment]
Has this problem been fixed in any of the newer versions of the SDK?
